tests/lib/cli-engine.js

```diff
@@ -21,6 +21,10 @@ const assert = require("chai").assert,
 
 const proxyquire = require("proxyquire").noCallThru().noPreserveCache();
 
+// Need use 'flatted' to parse the cache content as the 'file-entry-cache' use it to create cache
+// eslint-disable-next-line node/no-extraneous-require
+const flatted = require("flatted");
```

Comment:
We shouldn't use a dependency that we don't list in `dependencies` or `devDependencies`. This will make the code liable to break depending on which package manager is being used.